src/pkg/packager/deploy.go

```diff
@@ -370,6 +398,17 @@ func (p *Packager) deployComponent(ctx context.Context, component v1alpha1.ZarfC
 		return nil, fmt.Errorf("unable to run component after action: %w", err)
 	}
 
+	if len(component.HealthChecks) > 0 {
+		healthCheckContext, cancel := context.WithTimeout(ctx, p.cfg.DeployOpts.Timeout)
+		defer cancel()
+		spinner := message.NewProgressSpinner("Running Health checks for %s", component.Name)
```

Comment:
Lets not add more spinners tot things before we figure out how we want to deal with this long term.

Reply:
I see your point, but I do think we should give some feedback to the users that health checks are running. I'm fine with switching to a message.Infof() though it looks a bit strange since every other part of the deployment uses a spinner which leaves a different text color and has a check mark.

![image](https://github.com/user-attachments/assets/44f71e00-e41b-46d3-b38f-42188d8f8481)